<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUSBDlg</class>
 <widget class="QDialog" name="GUSBDlg">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure g.USBamp(s)</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>788</width>
     <height>990</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,1">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="configLayout">
      <item>
       <widget class="QPushButton" name="loadCfgButton">
        <property name="text">
         <string>Load Config</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveCfgButton">
        <property name="text">
         <string>Save Config</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="filters_layout">
      <item>
       <widget class="QComboBox" name="bandpass_comboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="bandpass_pushButton">
        <property name="text">
         <string>Apply Bandpass</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="notch_comboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="notch_pushButton">
        <property name="text">
         <string>Apply Notch</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="impedance_pushButton">
        <property name="text">
         <string>Refresh Impedances</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="devices_layout" rowstretch="1" columnstretch="1"/>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GUSBDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GUSBDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
